config/routes.rb

```diff
@@ -65,39 +65,46 @@
     get "/", to: "/conversions/projects#index"
     namespace :voluntary do
       get "/", to: "/conversions/voluntary/projects#index"
+      get "projects/:id", to: "/conversions/voluntary/projects#show", constraints: {id: /[0-9A-F]{8}-[0-9A-F]{4}-[0-9A-F]{4}-[0-9A-F]{4}-[0-9A-F]{12}/i}, as: :project
```

Comment:
Is it possible to extract the regex into a mehtod to give it some context, something lke `valid_uuid`, maybe?